packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java

```diff
@@ -0,0 +1,91 @@
+package org.alfresco.rest.rules;
```

Comment:
Add header ?

Reply:
Oh - that's interesting. We don't seem to be checking for headers on test files.  I'll add one anyway.

Reply:
I forgot to do this before merging, so I added it in a separate commit.